Cargo.toml

```
@@ -0,0 +1,14 @@
+[package]
+name = "aiocogeo-rs"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+byteorder = "1"
+bytes = "1.7.0"
+ndarray = "*"
+num_enum = "*"
+object_store = "0.10.2"
+thiserror = "1"
+tiff = "0.9"
+tokio = { version = "1.9", features = ["macros", "fs", "rt-multi-thread"] }
```

This is just for testing, correct? Some of these (`fs`, `rt-multi-thread`) [seem not WASM-compatible](https://docs.rs/tokio/latest/tokio/#wasm-support).
```suggestion

[dev-dependencies]
tokio = { version = "1.9", features = ["macros", "fs", "rt-multi-thread"] }
```

---

Yeah it's just for testing